endoreg_db/management/commands/import_video.py

```
@@ -222,6 +252,33 @@ def handle(self, *args, **options):
             self.stdout.write(self.style.ERROR("Failed to create VideoFile instance"))
             return
         
+        # Frame-level anonymization integration
+        if FRAME_CLEANING_AVAILABLE and video_file_obj.raw_file:
+            try:
+                self.stdout.write(self.style.SUCCESS("Starting frame-level anonymization..."))
+                # Properly instantiate FrameCleaner and ReportReader with correct arguments
+                frame_cleaner = FrameCleaner()
```

Pylint (3.3.7)

[error] Possibly using variable 'FrameCleaner' before assignment

(E0606)